app/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/claims/SelectNumberOfClaimsController.scala

```diff
@@ -109,15 +96,15 @@ class SelectNumberOfClaimsController @Inject() (
                   },
                   _ =>
                     formOk match {
-                      case Individual => Redirect(routes.EnterMovementReferenceNumberController.enterMrn())
-                      case Bulk       => Redirect(routes.EnterMrnMultiController.show())
+                      case Individual =>
```

Comment:
Might be better to have a ` toJourneyBindable` method somewhere instead so you don't need to repeat the entire Redirect line. I suspect it may also get used on other pages.

Reply:
This maybe confusing, but they are not the same.
`case Induvidual` -> The value in the form
`JourneyBindable.Single` -> PathBindable
I refactered it though!

Reply:
yeah, i wasn't saying use the same type - just that you should have a utility method to do the conversion rather than having three lines that repeat the same code to construct other parts of the same url.